README.md

```diff
@@ -1,32 +1,354 @@
+| D0b  | 267                                    |            |      | No             | ø                          |            ø             | [spine-generic](https://github.com/spine-generic/data-multi-subject)                            |
+| D1a  | 12                                     |    22.5    | 0.52 | Binary         | 100% Manual                |           0.51           | [D1a.tsv](https://github.com/ivadomed/model-spinal-rootlets/blob/main/dataset_creation/D1a.tsv) |
+| D1b  | 18                                     |    22.9    | 1.21 | Binary         | Prediction + Manual review |         0.52~0.6         | [D1b.tsv](https://github.com/ivadomed/model-spinal-rootlets/blob/main/dataset_creation/D1b.tsv) |
+|  D2  | 36 (20 from D0b + D1b) + 2 test images |    26.2    | 5.2  | Binary         | Prediction + Manual review |        0.65~0.75         | [D2.tsv](https://github.com/ivadomed/model-spinal-rootlets/blob/main/dataset_creation/D2.tsv)   |
+|  D3  | 31 + 2 test images                     |    26.5    | 5.5  | Level specific | Value modification         |         0.4~0.6          | [D3.tsv](https://github.com/ivadomed/model-spinal-rootlets/blob/main/dataset_creation/D3.tsv)   |
+
+<details>
+<summary>Details</summary>
+
+#### D1a)
+
+Dataset D1a was constructed with 12 subjects manually labeled (binary) from D0a. 12 MRIs from 6 subjects (3 female, 3
+male), each subject participated in 2 sessions, one with normal neck flexion and another with neck extension. The mean
+age is 22.5 y.o with a standard deviation of 0.52. Isotropic resolution of 0.6mm^3 (only one has a
+resolution of 0.7mm^3).
+
+One nnUNetV2 fold 3d_fullres model (M1a) was trained on D1a for 50 epochs, achieving a plateau with a dice
+score of approximately 0.51. M1a was used to predict 20 subjects from D0a (all head-normal and head-up
+images). After a manual review, two images were excluded because of unsatisfactory quality (sub-006_ses-headNormal and
+sub-009_ses-headNormal)
+
+> Refer to [issue#5](https://github.com/ivadomed/model-spinal-rootlets/issues/5).
+
+#### D1b)
+
+The resulting dataset, D1b consists of 18 MRIs from 10 subjects, 2 sessions (3 female, 7 male). The mean age is 22.9
+years old, with a standard deviation of 1.21. Isotropic resolution of 0.6mm^3 (only one has a resolution of 0.7mm^3).
+
+On this new dataset a five-fold training of nnUNetV2 3d_fullres model (M1b) has been conducted for 250 epochs, dice
+scores were between 0.52 and 0.6. An attempt was made to enhance results using the post-processing
+command of nnUNetV2, but no possible improvement was found so post-processing is useless in this case. Inference with
+M1b has been conducted on the full D0b (spine-generic) dataset.
+
+> Refer to [issue#7](https://github.com/ivadomed/model-spinal-rootlets/issues/7)
+
+#### D2)
```

Comment:
In https://github.com/ivadomed/model-spinal-rootlets/issues/7#issuecomment-1593539494 you mentioned:
> The images have been reoriented with sct_maths in LPI orientation.

If reorientation was applied to any image, it should also be mentioned here in this README.

Also, does the model M3 requires images in a specific orientation (e.g., `LPI`)? 
Or do you do reorientation by yourself before running inference (this is something we are doing in another project; see [here](https://github.com/ivadomed/model_seg_sci/blob/3757fad091f11f1556fe76037e6053826dd6a120/packaging/run_inference.py#L75) and [here](https://github.com/ivadomed/model_seg_sci/blob/main/packaging/run_inference.py#L182-L183))?

Reply:
I have modified the orientation when necessary but just notice that I have run Cadotte prediction on RPI orientation instead of LPI. I am going to test if the results are better with LPI orient.

Reply:
Result are exactly the same for prediction on RPI or LPI images